documentation/book/ref-operators-cluster-operator-configuration.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// assembly-operators-cluster-operator.adoc
+
+[id='ref-operators-cluster-operator-configuration-{context}']
+= Cluster Operator Configuration
+
+The Cluster Operator can be configured through environment variables.
+The following environment variables are supported:
+
+`STRIMZI_NAMESPACE`:: Required. A comma-separated list of namespaces that the operator should
+operate in. The Cluster Operator deployment might use the link:https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/#the-downward-api[{KubernetesName} Downward API^]
+to set this automatically to the namespace the Cluster Operator is deployed in. See the example below:
++
+[source,yaml,options="nowrap"]
+----
+env:
+  - name: STRIMZI_NAMESPACE
+    valueFrom:
+      fieldRef:
+        fieldPath: metadata.namespace
+----
+
+[[STRIMZI_FULL_RECONCILIATION_INTERVAL_MS]] `STRIMZI_FULL_RECONCILIATION_INTERVAL_MS`:: Optional, default: 120000 ms. The interval between periodic reconciliations, in milliseconds.
+
+`STRIMZI_LOG_LEVEL`:: Optional, default `INFO`.
+The level for printing logging messages. The value can be set to: `ERROR`, `WARNING`, `INFO`, `DEBUG` and `TRACE`.
+
+`STRIMZI_OPERATION_TIMEOUT_MS`:: Optional, default: 300000 ms. The timeout for internal operations, in milliseconds. This value should be
+increased when using {ProductName} on clusters where regular {ProductPlatformName} operations take longer than usual (because of slow downloading of Docker images, for example).
+
+`STRIMZI_DEFAULT_KAFKA_IMAGE`:: Optional, default `strimzi/kafka:latest`.
+The image name to use as a default when deploying Kafka, if
+no image is specified as the `Kafka.spec.kafka.image` in the xref:assembly-configuring-container-images-deployment-configuration-kafka[].
```

Comment:
We might want to remove the 'the' before the xref because this will render with the 'Section' label in front of it - and that seems strange. We might have to see this though before making the change here and in all of the following items.